bigbluebutton/bigbluebutton
# update(chat): add option to disable private chat

### What does this PR do?
- Adds `public.chat.private` to settings.yml, defaults to true
- Adds `privateChat` to disabledFeatures


### Motivation
Possibility to disable private chat.


### How to test

Disable private chat via settings.yml or disabledFeatures and start a meeting, users should be not able to start private chat.
